src/cppmangle.d

```diff
@@ -893,6 +893,13 @@ static if (TARGET_LINUX || TARGET_OSX || TARGET_FREEBSD || TARGET_OPENBSD || TAR
                 store(null);
             store(t);
         }
+
+        final const(char)* mangle_typeinfo(Dsymbol s)
+        {
+            buf.writestring("_ZTI");
```

Comment:
Correct, same goes for using ``_d_baseof`` on a C++ object.  I assume this is why your keeping around the wrapper for C++ typeinfo?